website/source/docs/builders/jdcloud.html.md

```diff
@@ -0,0 +1,88 @@
+---
+description: |
+    The `jdcloud` Packer builder helps you to build instance images
+    based on an existing image
+layout: docs
+page_title: 'JDCloud Image Builder'
+sidebar_current: 'docs-builders-jdcloud'
+---
+
+# JDCloud Image Builder
+
+Type: `jdcloud`
+
+The `jdcloud` Packer builder helps you to build instance images
+based on an existing image
+
+## Configuration Reference
+
+In order to build a JDCloud instance image, fullfill your configuration file. Necessary attributes
+are given below: 
+
+### Required-Parameters:
+
+- `type` (string) - This parameter tells which cloud-service-provider you are using, in our case, use 'jdcloud'
+- `image_id` (string) - New image is generated based on an old one, specify the base-image-id here. 
+- `access_key` (string) - Declare your identity , help us to know who you are. Alternatively you can set them as env-variable:`export JDCLOUD_ACCESS_KEY=xxx`
+- `secret_key` (string) - Same as access_key, write them this file, or set them as env-variable:`export JDCLOUD_SECRET_KEY=xxx`
+- `region_id` (string) - Region of your instance, can be 'cn-north-1/cn-east-1' etc.
```

Comment:
I would like to see a full list of available regions here, since they're hard-coded in the actual builder.